pico/tools/src/main/java/io/helidon/pico/tools/JavaC.java

```diff
@@ -231,7 +232,8 @@ public boolean hasWarnings() {
 
         public ToolsException maybeGenerateError() {
             if (!isSuccessful()) {
-                return new ToolsException("Creator compilation error");
+                return new ToolsException("Compilation error encountered:\n"
```

Comment:
This is an unrelated change, should not be part of this PR at all

Reply:
Discussed offline and agreed to keep this in.